autopilot/contractor/contractor.go

```diff
@@ -294,12 +294,17 @@ func (c *Contractor) performContractMaintenance(ctx *mCtx, w Worker) (bool, erro
 	// no subnet information, this was added to minimize churn immediately after
 	// adding 'subnets' to the host table
 	for _, h := range hosts {
-		if fcid, ok := hasContractInSet[h.PublicKey]; ok && len(h.Subnets) == 0 {
-			h.Subnets, _, err = utils.ResolveHostIP(ctx, h.NetAddress)
+		if fcid, ok := hasContractInSet[h.PublicKey]; ok && len(h.ResolvedAddresses) == 0 {
+			h.ResolvedAddresses, _, err = utils.ResolveHostIP(ctx, h.NetAddress)
 			if err != nil {
 				c.logger.Warnw("failed to resolve host IP for a host with a contract in the set", "hk", h.PublicKey, "fcid", fcid, "err", err)
 				continue
 			}
+			h.Subnets, err = utils.AddressesToSubnets(h.ResolvedAddresses)
```

Comment:
You don't need to populate `Subnets` anymore since that field won't be used or updated.

However, you need to update `autopilot/contractor/ipfilter.go` to make use of `ResolvedAddresses` instead of `Subnets`.

Reply:
Is there a reason the ipfilter used subnets instead of the IPs directly?  I tried looking at some old PRs but couldn't really find anything.